src/pipeline/pose_engine.py

```diff
@@ -90,9 +92,34 @@ def get_input_tensor_shape(self):
         """
         return self._tfengine.input_details[0]['shape']
 
-    def parse_output(self, heatmap_data, offset_data, threshold):
+    
+    def parse_output_Movenet(self, keypoints_with_scores, height, width):

```

Comment:
@bhavikapanara let's move out the model specific code into its own pyhton modules/classes that return a uniform format of keypoints and scores given a sample. 

Fixing bugs in individual models (posenet, movenet, blazepose, etc.) should not require changes in the higher level APIs. Also adding a new model should also ideally not require changes in the higher level API code.

For example we can use an [abstract base class](https://docs.python.org/3/library/abc.html) that's initialized with a configuration set and offers a run() method for single sample inferences:

```
AbstractPoseModel:

__init__(model_config)

run(img: Image) -> kps, template_image, thumbnail

```

Reply:
Done